### DISCUSSION TITLE
busybox 1.29 fuer developer wieder aufnehmen fuer quicktests

### DISCUSSION THREAD

**Discussion by Himan2001** [Freetz-NG/freetz-ng#561] [Unerfüllte Wünsche]
Frage:

koennen wir gerade bzgl. der Problematik bei neuen Boxen wie der 55xx Serie zu Testzwecken im Developer-Modus
die "original" AVM busybox Versionen 1.29.x zum testen wieder in die "einfache" Auswahl aufnehmen. Da die serielle Schnittstelle
im direkten Vergleich auch nicht immer wirklich alles ausgibt, könnte ein schneller Downgrade zu Testzwecken bei neueren
Geräten evtl. auch schneller zur Lösungsfindung bei Problemen+Treibern führen ?

**Comment by fda77** [comment]
Mach die dir doch einfach selbst rein: 2 Dateien editieren, Patches hinzufügen und generier-script ausführen
